packages/react/collection/src/Collection.stories.tsx

```diff
@@ -157,7 +157,7 @@ const MemoItem = React.memo(Item);
 const MemoItems = React.memo(WrappedItems);
 
 function LogItems({ name = 'items' }: { name?: string }) {
-  const getItems = useCollection({});
+  const getItems = useCollection(undefined);
```

Comment:
Basically, we were passing `scope={undefined}` everywhere on the component props, but here we were passing an empty object.